app/assets/stylesheets/components/navigation.scss

```diff
@@ -112,4 +112,26 @@
       }
     }
   }
+
+  &--item {
+    padding: $su-3;
+    text-align: center;
+    width: 100%;
+
+    &.selected {
+      border-bottom: 4px solid var(--accent-brand);
+
+      .crayons-tabs--label {
```

Comment:
`__` instead of `--` since [it's a child element, not a modifier](http://getbem.com/naming/).

Reply:
You caught me not knowing BEM and guessing 😂

Reply:
Resolved in latest commits.